libshaderc/src/shaderc_cpp_test.cc

```diff
@@ -504,6 +505,123 @@ TEST_F(CppInterface, PreprocessingOnlyModeSecondOverridesDisassemblyMode) {
               HasSubstr("void main(){ }"));
 }
 
+// To test file inclusion, use hashmap as a fake file system to store fake files
```

Comment:
Should say how the unordered map describes a filesystem:    The unordered map represents a filesystem by  mapping  filename string to the contents of that file as a string.